scripts/analyze_provider_raw.py

```
@@ -0,0 +1,89 @@
+            series_id = series_info.get('series_id')
+            name = series_info.get('name')
+            
+            # print(f"Checking series: {name}...")
+            
+            detailed_info = client.get_series_info(series_id)
+            episodes = detailed_info.get('episodes', {})
+            
+            # Episodes might be list or dict
+            all_eps = []
+            if isinstance(episodes, dict):
+                for season, eps in episodes.items():
```

Ruff (0.14.7)

Loop control variable `season` not used within loop body

Rename unused `season` to `_season`

(B007)